src/Assets/Asset.php

```diff
@@ -934,11 +934,13 @@ public function maybe_get_min_file( $url ) {
 			}
 		}
 
-		if ( ! $script_debug ) {
+		if ( Config::should_ignore_script_debug() || ! $script_debug ) {
```

Comment:
In the case of `Config::should_ignore_script_debug() => true` the same logic is running twice. Do we really need this in the same logic in 2 places? 
z
I see you have added the array_unique as well to avoid duplicates. But if we only kept this one the result should be same, right? 

@bordoni